db/migrate/20210428175951_add_resolved_by_id_to_flags.rb

```diff
@@ -0,0 +1,5 @@
+class AddResolvedByIdToFlags < ActiveRecord::Migration[6.1]
+  def change
+    add_column :flags, :resolved_by_id, :integer, default: 0, null: false
```

Comment:
Sorry I missed this before, can we remove the default on this and allow it to be null please?

Reason being, If we leave the default as 0 and  add a resolved_by association to the flags model at some point in future it will try and find a user with an id of 0 and cause an exception when that user can't be found.

Whereas if we allow it to be null the association will just return nil which will be expected when there is no resolver.

Reply:
Oh no worries, I was actually confused on this... I'm still somewhat shaky with migrations and databases. I'm glad you brought this up because now I understand better when to allow null. I think it should all be good now.